src/sentry/tasks/store.py

```diff
@@ -137,6 +136,11 @@ def _do_preprocess_event(
             "organization", Organization.objects.get_from_cache(id=project.organization_id)
         )
 
+    if data["platform"] in ("javascript", "node"):
+        from sentry.lang.javascript.processing import get_symbolication_function
+    else:
+        from sentry.lang.native.processing import get_symbolication_function
```

Comment:
IMO this is a fair thing to do right now

Though I get the original Idea of plugins and Event processors / enhancers of being able to process mixed stack traces. However I believe we should re-think that idea/concept more broadly taking into account the current status of things.